src/pages/index.tsx

```diff
@@ -18,6 +19,9 @@ const Home = () => {
       <p>
         Hejka {query.data?.first_name} {query.data?.last_name}
       </p>
+      <Link href="/plans">
+        <button>Go to Plans</button>
```

Comment:
to nie jest przycisk, skoro jest linkiem

https://medium.com/@h_locke/links-vs-buttons-e8b523660fb3